api/src/pcapi/routes/backoffice/templates/offer/list.html

```diff
@@ -136,7 +136,11 @@ <h2 class="fw-light">Offres individuelles</h2>
                     </ul>
                   </div>
                 </td>
-                <td>{{ offer.id }}</td>
+                {% if has_permission("READ_OFFERS") %}
+                  <td>{{ links.build_offer_details_link(offer) }}</td>
+                {% else %}
+                  <td>{{ offer.id }}</td>
+                {% endif %}
```

Comment:
`READ_OFFERS` est déjà la permission nécessaire pour voir cette liste, donc la condition est forcément vérifiée.
```suggestion
                <td>{{ links.build_offer_details_link(offer) }}</td>
```